### ISSUE TITLE
Add support for edge-security-policy param in google_compute_backend_service

### ISSUE THREAD

**Issue by pedromiranda-telus** [hashicorp/terraform-provider-google#12867]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment. If the issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If the issue is assigned to a user, that user is claiming responsibility for the issue. If the issue is assigned to "hashibot", a community member has claimed the issue already.

### Description

Currently, the resource `google_compute_backend_service` has no any option to set [--edge-security-policy](https://cloud.google.com/sdk/gcloud/reference/compute/backend-services/update#--edge-security-policy).

### New or Affected Resource(s)

* google_compute_backend_service

### Potential Terraform Configuration

```tf
# Propose what you think the configuration to take advantage of this feature should look like.
# We may not use it verbatim, but it's helpful in understanding your intent.
resource "google_compute_backend_service" "esports" {
  (...)
  edge_security_policy = google_compute_security_policy.my_security_policy.id
}
```

### References

N/A.

**Comment by sanghaniJ** [comment]
I've gone through this issue and the request is regarding support for a new field `edge_security_policy` in the resource `compute_backend_service`. 
- While going through the schema, I found a similar field`securityPolicy` present. 
- I noticed both edgeSecurityPolicy and securityPolicy are [Output only] as per [API Docs](https://cloud.google.com/compute/docs/reference/rest/v1/backendServices)
- I tried adding the field into the schema with `output:true` i.e Computed considering API Docs, but after being created the field is returned empty in the state file.
- So, either the docs needs to be updated or the behavior of edgeSecurityPolicy is similar to securityPolicy and it needs to be set optional with code where it is set when not done set by Create/Update. The same in code for securityPolicy is mentioned [here](https://github.com/hashicorp/terraform-provider-google/blob/71914fdf37bf16d77d55774f2e76c6f8b7de04d7/google/resource_compute_backend_service.go#L1297)
